lib/mail_room/mailbox_watcher.rb

```diff
@@ -114,6 +114,8 @@ def run
       @running = true
 
       self.idling_thread = Thread.start do
+        process_mailbox
```

Comment:
Well, all of this mail _will_ be processed because it's picked up by `search(UNSEEN)`, the first time `process_mailbox` is called after `idle` stops blocking when the first _new_ email comes in.

Adding this line is simply removing that waiting period.